label_studio/tasks/migrations/0009_auto_20210913_0739.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.1.13 on 2021-09-13 07:39
+
+from django.db import migrations
+from django.contrib.postgres.operations import TrigramExtension
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tasks', '0008_auto_20210903_1332'),
+    ]
+
+    operations = [
+        TrigramExtension(),
+        migrations.RunSQL(
```

Comment:
Could you create the similar migration for task.data please?

Reply:
Discussed and decided to make more investigation before doing that.